desc/magnetic_fields/_core.py

```diff
@@ -2621,17 +2623,29 @@ def field_line_integrate(
         Size to split computation into chunks of evaluation points.
         If no chunking should be done or the chunk size is the full input
         then supply ``None``. Default is ``None``.
+    iota: bool
+        Whether or not to also find the rotational transform of each field line. If
+        True, requires axis to be passed in. iota is computed assuming the poloidal
+        angle is increasing counter-clockwise about the axis as the field line
+        moves along increasing toroidal angle phi.
+    axis: Curve
+        The magnetic axis relative to which the rotational transform is computed. Only
+        required if iota=True.
     kwargs: dict
         keyword arguments to be passed into the ``diffrax.diffeqsolve``
 
     Returns
     -------
-    r, z : ndarray
+    r, z : ndarray of shape [n_phis, n_initial_points]
         arrays of r and z coordinates of the field line, corresponding to the
         input phis
+    iotas : 1-D ndarray of shape [n_initial_points]
+        if iota=True, the rotational transform computed for each field line traced.
     """
-    r0, z0, phis = map(jnp.asarray, (r0, z0, phis))
+    r0, z0, phis = map(jnp.atleast_1d, (r0, z0, phis))
```

Comment:
Had to change this to always be 1D, otherwise the grabbing of the last theta value later will throw an index error when r0 and z0 are passed in as only 0-D arrays

Reply:
ie the whole block would be
```python 
r0, z0, phis = map(jnp.asarray, (r0, z0, phis))
assert r0.shape == z0.shape, "r0 and z0 must have the same shape"
errorif(iota and axis is None, ValueError, "Must pass in an axis if iota=True")
rshape = r0.shape
r0 = jnp.atleast_1d(r0).flatten()
z0 = jnp.atleast_1d(z0).flatten()